frontend/packages/dev-console/src/templates/index.ts

```diff
@@ -1 +1,5 @@
 export * from './pipeline';
```

Comment:
I wonder if we should just rename this to `pipelines` and put everything in here. Each with a single (non-default) export seems a bit weird.

Reply:
Agree, we can put everything in one file.

Reply:
Done.